lib/logical/export.js

```diff
@@ -8,6 +8,8 @@ const StructureDefinition = require('./StructureDefinition');
 
 /** @typedef {import('./ElementDefinition')} ElementDefinition */
 
+// TODO: Update for DSTU2
```

Comment:
Do we need to implement this before merging, or is this something that we're comfortable leaving as TODO?

Reply:
We're comfortable merging it as-is.  For this first release, DSTU2 exports just won't support logical models.  I had actually forgotten I still needed to do this until I put together the PR -- then I remembered.  But still...  I think the team would rather have the working profile, extension, value set exporters in master now than wait for logical model support.